tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/CiamIntegrationTests.cs

```diff
@@ -87,11 +82,9 @@ public async Task ROPC_Ciam_Async(string authorityFormat, int authorityVersion)
         }
 
         [TestMethod]
-        [DataRow("https://{0}.ciamlogin.com/", 0)] //https://tenantName.ciamlogin.com/
-        [DataRow("https://{0}.ciamlogin.com/{1}.onmicrosoft.com", 1)] //https://tenantName.ciamlogin.com/tenantName.onmicrosoft.com
-        [DataRow("https://{0}.ciamlogin.com/{1}", 2)] //https://tenantName.ciamlogin.com/tenantId
-        public async Task ClientCredentialWithClientSecret_Ciam_Async(string authorityFormat, int authorityVersion)
+        public async Task ClientCredentialWithClientSecret_Ciam_Async()
```

Comment:
Test naming is incorrect since cert is used. Suggesting maybe to move the custom domain test into a separate test?

Reply:
Changed it to `ClientCredential` which is more generic